### PR TITLE
Added new info about tags, edited tiles to cards, other small edits

### PR DESCRIPTION
**Pull request by dayleparker**
Added new info about tags, edited the term 'tiles' to 'cards' (following a Slack conversation between UXD & CCS), other small edits like a reminder step to review content against the Best Practices list.

Jira: https://issues.redhat.com/browse/HCCDOC-3142 

@Hyperkid123 would you mind reviewing to catch any errors in the tags section? Also I want to separate the sections for each tag a bit better, any ideas how to improve the markdown?

I will request a review from a content creator as well to make sure the updates make sense.

Thank you!

### CODE REVIEW

**Review by Hyperkid123 — COMMENTED** [review]
Seems great.

File: docs/quickstarts/README.md
Comment (CodeDuchess) [inline docs/quickstarts/README.md:55]: It would be helpful to know how long it will take to get your QS merged. It can take anywhere from 1 day to 3 weeks in my experience, so it would be good to get an estimation from the team.

File: docs/quickstarts/README.md
Comment (CodeDuchess) [inline docs/quickstarts/README.md:53]: who should we @ in the actual PR? Also, Martin told me to go into the forum-consoledot-UI Slack channel, and alert _@crc-experience-services-team_ and request that my PR be merged. He said this was due to his reduced bandwith. So, I think we should add that to the README.

File: docs/quickstarts/README.md
Comment (CodeDuchess) [inline docs/quickstarts/README.md:51]: add the colleagues that need to be @ on this work.

File: docs/quickstarts/README.md
Comment (CodeDuchess) [inline docs/quickstarts/README.md:165]: "work with engineering to add the content to the HCC." What exactly does this mean? I think we should spell this out for new users.

**Review by CodeDuchess — COMMENTED** [review]
Looks good, Dayle just a few comments.

File: docs/quickstarts/README.md
Comment (dayleparker) [reply docs/quickstarts/README.md:55]: Good question - @Hyperkid123 can you advise on an estimate to provide here?

File: docs/quickstarts/README.md
Comment (dayleparker) [reply docs/quickstarts/README.md:53]: @Hyperkid123 do we want to mention any specific team members, besides @crc-experience-services-team ? Is it all right to mention our internal Slack channel/tag in this public repo?

File: docs/quickstarts/README.md
Comment (Hyperkid123) [reply docs/quickstarts/README.md:55]: It's tough to give an estimate, it depends on the services team and how quickly they can get to the PR. But as I have mentioned, I am open to adding a person from the docs team to the project with merge rights if that would help with the velocity.

Comment (dayleparker) [reply docs/quickstarts/README.md:55]: Since the timing is variable, I'm going to leave off this info for now - we can add it later as needed.

File: docs/quickstarts/README.md
Comment (Hyperkid123) [reply docs/quickstarts/README.md:53]: In the PR you want to tag @RedHatInsights/platform-experience-services as a reviewer, I will setup the Repo so this is done automatically.

Comment (dayleparker) [reply docs/quickstarts/README.md:53]: Thank you for setting that up automatically, @Hyperkid123!

File: docs/quickstarts/README.md
Comment (dayleparker) [reply docs/quickstarts/README.md:51]: After discussing with the PMs, that list of contacts will remain in the Gdoc/Confluence page as this is a public repo https://docs.google.com/document/d/1KZfvvxtUzswojI5XfCgoqfEJnnRaLyl5xGlev6Bm-Rs/edit?tab=t.0#heading=h.jjeosz2sc2y7

File: docs/quickstarts/README.md
Comment (dayleparker) [reply docs/quickstarts/README.md:165]: Reworded a little - basically this meant 'continue to the next steps where you work together with engineering to merge your work'

**Comment by Hyperkid123** [comment]
Sorry about the CI failures. Konflux has been having some issues in the last few days

**Review by florkbr — APPROVED** [review]
LGTM :shipit: 

@JetyAdam FYI

**Comment by dayleparker** [comment]
Thank you for approving it, @florkbr! Can you please merge it as well? (It seems to be blocked by some auto checks)
Let me know if there's anything I need to do from my side.

**Pull request merged**
